components/board/analog_smoother.go

```diff
@@ -101,13 +97,12 @@ func (as *AnalogSmoother) Start(ctx context.Context) {
 		as.data = utils.NewRollingAverage(numSamples)
 		nanosBetween = 1e9 / as.SamplesPerSecond
 	} else {
-		as.logger.CDebug(ctx, "Too few samples to smooth over; defaulting to raw data.")
+		as.logger.Debug("Too few samples to smooth over; defaulting to raw data.")
```

Comment:
This strikes me as an undesirable change, but I don't have any `Context` to use any more. `Start` is called from `SmoothAnalogReader`, which does not have any `Context`s. I could change `SmoothAnalogReader` to take one as an argument, but then this change would touch half a dozen different files, which feels wrong. Any advice? (I still don't know why you'd want a context in your logging statements, so don't have an informed opinion about these things.)